modules/API/pages/built-in-endpoints.adoc

```diff
@@ -2915,7 +2658,7 @@ Request::
 --
 [source.wrap,console]
 ----
-curl --user tigergraph:tigergraph -X GET 'http://localhost:8123/gsql/queries/install/1234'
+curl --user tigergraph:tigergraph -X GET 'http://localhost:14240/gsql/v1/queries/install/1234'
```

Comment:
BTW, have we changed all 8123 to 14240 as well?

Reply:
We missed some places, let me change them

Reply:
Done